_data/theFAQs.json

```diff
@@ -151,102 +146,102 @@
          "answer": "<p>Currently, moving your data from one storage provider to another is not supported via the UI. You can do this move with assistance from Blockstack.</p>"
       },
       {
-         "category": "dappdevs",
```

Comment:
The new category seems not to be visible: https://deploy-preview-612--docsproduction.netlify.app/faqs/allfaqs#dapp-developers-questions

Maybe the category id is used to filter and show these in the page?

Reply:
Oops, this was the result of a sloppy search/replace on my part. I'll fix.